activerecord/test/cases/fixtures_test.rb

```diff
@@ -1399,13 +1399,6 @@ class MultipleDatabaseFixturesTest < ActiveRecord::TestCase
       rw_conn = ActiveRecord::Base.connection
       ro_conn = ActiveRecord::Base.connection_handlers[:reading].connection_pool_list.first.connection
 
-      assert_not_equal rw_conn, ro_conn
-
-      enlist_fixture_connections
-
-      rw_conn = ActiveRecord::Base.connection
-      ro_conn = ActiveRecord::Base.connection_handlers[:reading].connection_pool_list.first.connection
```

Comment:
Why did you delete the test that ensures this works? If this test isn't correct can you create a test that is more correct then?

Reply:
This part of the test is no longer necessary. The test is still checking if both pools are the same between replica->master https://github.com/rails/rails/pull/37774/files#diff-9a795b6442abbee1bf369da52a59961cR1402

What I deleted is the check to make sure both pool aren't the same which is now useless.